### DISCUSSION TITLE
Sonoff SV - Garage + temperature + reed switch

### DISCUSSION THREAD

**Discussion by mickelk** [RavenSystem/esp-homekit-devices#1693] [MEPLHAA Scripts]
Hi!
I am using Sonoff SV for my garage door, I found JSON code like this:

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12,"v":1,"i":0.5}]},"1":{"a":0},"2":{"r":[{"g":12,"i":1.5},{"g":12,"v":1,"i":0.5},{"g":12,"v":1,"i":2}]},"3":{"a":2},"t":40,"d":33,"f3":[{"g":14,"t":0}],"f4":[{"g":14}],"f2":[{"g":5,"t":0}],"f5":[{"g":5}]},{"t":22,"s":5,"g":3,"n":3,"j:":15,"z":0.1}]}

Sonoff SV + reed switch - Garage door + DS18B20 temperature sensor

The temperature works, unfortunately the problem is when closing the garage. a few seconds after clinching, close - sonoff gives a second beep and the gate stops. I have the "Opening" status on my phone, is the problem with JSON?

In the code, I only changed the time in seconds to close the gate - d: 33

When using the "no thermometer" code everything works perfectly:

{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"t":40,"d":33,"e":5,"0":{"r":[{"g":12,"v":1,"i":1.5}]},"1":{"r":[{"g":12,"v":1,"i":1.5}]},"2":{"r":[{"g":12,"v":1,"i":1.5}]},"3":{"r":[{"g":12,"v":1,"i":1.5}]},"f3":[{"g":14,"t":0}],"f4":[{"g":14}]}]}

**Comment by i3laze** [comment]
The second beep comes when you switch the direction of the movement:
Closing -> Opening or Opening -> Closing will make two clicks.
When the state is Closed or Opened, then HAA should produce one click.

Your current Opening->Open duration is **d:33** seconds, is that value relevant?

**Reply by mickelk** [reply]
I managed to do, leaving the json code may be useful to someone
yes: d: 33 it is the gate closing time, there is only one 100% close connector 

`
{"c":{"l":13,"b":[{"g":0,"t":5}]},"a":[{"0":{"r":[{"g":12,"v":1,"i":1.5}]},"1":{"r":[{"g":12,"v":1,"i":1.5}]},"2":{"r":[{"g":12,"v":1,"i":1.5}]},"3":{"r":[{"g":12,"v":1,"i":1.5}]},"t":40,"d":32,"e":5,"f3":[{"g":14,"t":0}],"f4":[{"g":14}],"s":0,"b":[]},{"t":22,"b":[],"n":3,"g":3,"z":0.1}]}`
